bin/tick.sh

```diff
@@ -5,7 +5,7 @@
 . services/ir/.ir.env
 
 # NeoGo binary path.
-NEOGO="${NEOGO:-docker exec -it main_chain neo-go}"
```

Comment:
Roll it back please and set `NEOGO` in the environment where you set up devenv instead